src/main/java/com/cburch/logisim/gui/generic/AttrTable.java

```diff
@@ -49,6 +48,11 @@
 import javax.swing.table.TableModel;
 
 @SuppressWarnings("serial")
+
+/*
+ * Attribute table panel
+ * Shows detailed attributes of selected element.
```

Comment:
Fix wording: "Shows detailed attributes of the selected element."

Reply:
Done.